pnet_sys/src/lib.rs

```diff
@@ -96,88 +90,3 @@ fn htons(u: u16) -> u16 {
 fn ntohs(u: u16) -> u16 {
     u16::from_be(u)
 }
-
-fn make_in6_addr(segments: [u16; 8]) -> In6Addr {
```

Comment:
How come these have been moved?

Reply:
These were split up and moved as libc::in6_addr is a different struct with different fields/nested-fields than winapi-rs 0.3. winapi-rs 0.3 is more explicit with its handling of unions so instead of just calling it a DWORD for example, it calls it a struct with a field that is named Word, and accessed via a method. These methods are marked as unsafe. See here: https://docs.rs/winapi/0.3.6/winapi/shared/in6addr/struct.in6_addr_u.html.   I figured splitting them into OS specific files made more sense, feel free to correct me.